components/event-bus/Makefile

```diff
@@ -63,14 +63,15 @@ clean:
 
 .PHONY: test
 
-vendor: Gopkg.lock Gopkg.toml
+vendor: Gopkg.lock
 	@echo '+ Pulling vendored dependencies'
 	@dep ensure -v --vendor-only
 
 
 GOTEST_FLAGS := -cover
 test-local: vendor
 	@echo "+ Performing unit tests"
+	$(eval testpkgs := $(shell go list -test -f '{{ if .TestGoFiles }}{{ .ForTest }}{{end}}' ./... ))
```

Comment:
Is the `eval` call really needed? Can't we just have the variable defined like any other variable outside of the function since it technically doesn't clash with anything?

Reply:
this target is evaluated in the docker container. We need this as we need go and currently we dont have go on the machine that executes the Makefile